supportedBackends/google/pipelines/common/src/test/scala/cromwell/backend/google/pipelines/common/PipelinesApiAsyncBackendJobExecutionActorSpec.scala

```diff
@@ -1917,14 +1924,14 @@ class PipelinesApiAsyncBackendJobExecutionActorSpec
 
   it should "call GroupMetricsActor when job runs into cloud quota delay" in {
     val papiBackend = setupBackend
-    papiBackend.checkAndRecordQuotaExhaustion(RunStatus.AwaitingCloudQuota)
+    papiBackend.checkAndRecordQuotaExhaustion(RunStatus.AwaitingCloudQuota(Seq.empty))
 
     mockGroupMetricsActor.expectMsgType[RecordGroupQuotaExhaustion]
   }
 
   it should "not call GroupMetricsActor when job hasn't run into quota delay" in {
     val papiBackend = setupBackend
-    papiBackend.checkAndRecordQuotaExhaustion(RunStatus.Running)
+    papiBackend.checkAndRecordQuotaExhaustion(RunStatus.Running(Seq.empty))
```

Comment:
You could, if you choose, give this arg a default value of empty list and not need to make all these `(Seq.empty)` changes.